.github/workflows/on_master_commit.yml

```diff
@@ -0,0 +1,60 @@
+name: master_commit
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  commit_master:
+    name: commit to master
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '3.0.5'
+          channel: 'stable'
+      - name: ApiKeysFlutter
+        run: |
+          touch "keys_prod.json";
+          echo "${{ secrets.KEYS_PROD }}" >> keys_prod.json;
+          touch "keys_sandbox.json";
+          echo "${{ secrets.KEYS_SANDBOX }}" >> keys_sandbox.json;
```

Comment:
do we need to create key files for all build types to build only sandbox/prod?

Reply:
u right, here we can skip creating of keys_prod